scripts/jenkins/ardana/ansible/roles/ardana_qe_tests/templates/tests/neutron.sh.j2

```diff
@@ -25,7 +27,18 @@ fi
 if $OCTAVIA; then
     export OCTAVIA=true
 fi
-cd {{ ardana_qe_tests_dir }}/ardana-qa-tests/neutron/sdk_scripts/
+cd {{ ardana_qe_test_venv }}
+git clone https://github.com/vmware/vsphere-automation-sdk-python.git 
```

Comment:
We should not run code from external on internal hardware without having control. Same for pip.

Reply:
Should that become an unsupported package in the cloud product?

Reply:
OK, let me find a different way to validate resources on vcenter and remove this external dependency. Thanks.

Reply:
Probably doable as vsphere-automation-sdk-python is an opensource project but I believe, that will require some maintenance. 
 vSphere’s API Explorer (Part of the vcenter product) will be better option as It will not require any dependency and maintenance. Just need to update test codes & wrapper scripts.